src/spikeinterface/postprocessing/amplitude_scalings.py

```diff
@@ -457,29 +497,37 @@ def fit_collision(
 ):
     """
     Compute the best fit for a collision between a spike and its overlapping spikes.
-    The function first cuts out the traces around the spike and its overlapping spikes, then
-    fits a multi-linear regression model to the traces using the centered templates as predictors.
+
+    The problem addressed here is to compute the scaling factor of a waveform
+    to its unit template in the presence of temporally and spatially
+    colliding spikes (i.e. spikes that occur close to a given spike). When
+    the waveform of a spike overlaps with other, colliding spikes, these
+    colliding spikes will contribute to the spike amplitude.
+
+    This is addressed by fitting a multivariate regression. `y` is
+    the observed waveform (including the spike of interest and colliding spikes).
+    `X` is the corresponding set of unit templates, with each template
+    temporally shifted to match the position of its associated spike in the
+    original waveform. The regression coefficients represent the scaling
+    factors applied to each template to best match the waveform.
```

Comment:
Mmm that's a good point. Not sure. Good idea to discuss it on the docstrings hackaton discussion : )

Reply:
Yes I agree this would be great to discuss it is a very interesting feature. Maybe we could have a general issue on style-related questions for posting discussions such as the the ones raised in the PR until we get around to addressing them?

For me the latex-styling is really cool but makes the docstring itself hard to read, whereas not having the latex makes the docs look less nice but at least the docstrings are still easily readable. So on the possibilities for (docstring, docs) of (hard to read, awesome to read) vs (okay to read, okay to read) I tend to the latter.

Reply:
I agree with Joe on both points. Unless we move over to do docstring testing/other crazy doctoring format I would prefer to be okay on both rather than bad on one and good on the other.

Reply:
For my own bias I rather make the docstring readable as a developer. Most of the time I am reading the code on an editor and not on a webpage.

Reply:
Thanks both I'll leave this for now then